pkg/specgen/generate/container_create.go

```diff
@@ -245,9 +245,12 @@ func createContainerOptions(ctx context.Context, rt *libpod.Runtime, s *specgen.
 		if err != nil {
 			return nil, err
 		}
-		if newWD != "" {
-			options = append(options, libpod.WithCreateWorkingDir())
-		}
+		s.WorkDir = newWD
+	}
+	if s.WorkDir == "" {
+		s.WorkDir = "/"
+	} else {
+		options = append(options, libpod.WithCreateWorkingDir())
```

Comment:
I thought we decided to only do this for entrypoint specified by the image - if the user specifies `/ect` accidentally we would still error on that?

Reply:
Ok then I need to remove the test.